### ISSUE TITLE
Error when sorting bags

### ISSUE THREAD

**Issue by halfreak** [arkayenro/arkinventory#1133]
Hi there! Ever since r803 I'm having errors when sorting bags. This log is from r806:

`22x ArkInventory\ArkInventory-30821.lua:7619: attempt to index local 'frame' (a nil value)
ArkInventory\ArkInventory-30821.lua:7619: in function `Frame_Item_Update_Border'
ArkInventory\ArkInventory-30821.lua:2735: in function `ObjectLockChanged'
ArkInventory\ArkInventoryStorage.lua:504: in function `?'
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua:119: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:119>
[C]: ?
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua:29: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-7.lua:64: in function `Fire'
Ace3\AceEvent-3.0\AceEvent-3.0-4.lua:120: in function <Ace3\AceEvent-3.0\AceEvent-3.0.lua:119>
[C]: in function `resume'
ArkInventory\ArkInventory-30821.lua:11158: in function `ThreadResume'
[string "*:OnUpdate"]:1: in function <[string "*:OnUpdate"]:1>`

**Comment by arkayenro** [comment]
sorting as in restack? or something else?

**Comment by halfreak** [comment]
Yes, restack. Sorry, didn't know exactly how it's called in English.

**Comment by arkayenro** [comment]
its a bank slot that was locked and its trying to update that frame so if youre not at the bank then thats really weird

**Comment by halfreak** [comment]
OK, well the output is pretty long, so I don't know the means of copying that much text from WoW. Besides, it's all about the same. It starts with:
`[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 8, 22 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 8, 23 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 8, 24 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 8, 25 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 8, 27 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 8, 28 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 1 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 2 ) - nil was not found`

Then somewhere around the middle it does that:
`[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 80 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 81 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 82 ) - nil was not found
[18:07:53] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 83 ) - nil was not found
[18:07:53] ArkInventory: Сортировка: Рюкзак - Выполнено
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 1, 1 ) - nil was not found
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 1, 2 ) - nil was not found
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 1, 3 ) - nil was not found
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 1, 4 ) - nil was not found`

And goes all the way to:
`[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 80 ) - nil was not found
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 81 ) - nil was not found
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 82 ) - nil was not found
[18:07:54] ArkInventory: WARNING> Code Failure: ObjectLockChanged( 3, 9, 83 ) - nil was not found`

**Comment by arkayenro** [comment]
try [r808-alpha](https://www.wowace.com/projects/ark-inventory/files/2661210)
